### PR TITLE
Configure CSS linter and apply

### PR DESCRIPTION
**Pull request by clementbiron**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -9,12 +9,12 @@ on:
...
+  test:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
           cache: npm
       - run: npm ci
-      - run: npm run lint:markdown
+      - run: npm run test
```
Comment (MattiSG) [inline .github/workflows/test.yml:20]: Having independent steps could make CI results easier to read 🙂 
```suggestion
      - run: npm run lint:markdown
      - run: npm run lint:css
```

File: package.json [hunk 2]
```diff
@@ -13,11 +13,14 @@
   "license": "CC BY-SA 4.0",
   "author": "Open Terms Archive",
   "scripts": {
-    "test": "npm run lint:markdown",
-    "lint:markdown": "markdownlint *.md content/**/*.md"
+    "test": "npm run lint:markdown && npm run lint:css",
+    "lint:markdown": "markdownlint *.md content/**/*.md",
+    "lint:css": "./node_modules/.bin/stylelint \"themes/opentermsarchive/assets/css/*.css\""
```
Comment (MattiSG) [inline package.json:18]: I'm surprised you need to explicitly use `./node_modules/.bin/`. NPM scripts should also wrap bin location. Did you try simply using:
```suggestion
    "lint:css": "stylelint themes/opentermsarchive/assets/css/*.css"
```
? 🙂

**Review by MattiSG — APPROVED** [review]
Very welcome! 👍

File: package.json (hunk 2 above)
Comment (clementbiron) [reply package.json:18]: It's not necessary to explicity use `./node_modules/.bin/`. I thought it was a good practice, I correct

**Review by MattiSG — APPROVED** [review]
Already approved  :)

**Comment by clementbiron** [comment]
Sorry, `lint_markdown` was configured as a necessary check is the repo conf (which duplicated the CI workflow) and it prevented me from merging this PR.

**Pull request merged**